chia/farmer/farmer.py

```diff
@@ -699,6 +702,7 @@ async def _periodically_clear_cache_and_refresh_task(self) -> None:
                     for key, add_time in self.cache_add_time.items():
                         if now - float(add_time) > self.constants.SUB_SLOT_TIME_TARGET * 3:
                             self.sps.pop(key, None)
+                            self.sp_hash_to_filter_size.pop(key, None)
```

Comment:
it looks like we could save a fair amount of space by combining all these dicts into one. but that's for a future improvement